INTEGRATING.md

```diff
@@ -0,0 +1,14 @@
+## Integrating Prometheus
+
+Prometheus integrates with the following
+
+1. Any charm that supports the `prometheus_scrape` interface in
+serving as a scrape target for Prometheus. This integration is done
+using the Prometheus Charm library. The documentation of the charm
+library, available through
+[Charmhub](https://charmhub.io/prometheus-k8s) provides further
+details on how this integration may be done.
+
+2. Grafana
```

Comment:
This needs a follow-up PR to document how.

Reply:
Indeed. ATTN @rbarry82 .

Reply:
I can work with @rbarry82 to ensure this is picked up in a future PR.

Reply:
As part of the `grafana-dashboard` work, I'll add a PR here which uses the Grafana consumer library.